_health-care/_js/components/form-elements/ErrorableRadioButtons.jsx

```diff
@@ -57,15 +57,15 @@ class ErrorableRadioButtons extends React.Component {
       }
       const checked = storedValue !== undefined && optionValue === storedValue ? 'checked=true' : '';
       return (
-        <div key={reactKey++} className="radio-buttons-inline">
+        <div key={reactKey++} className="hca-radio-buttons">
           <input
               checked={checked}
-              id={optionValue}
-              name={this.inputId}
+              id={`${this.inputId}-${index}`}
+              name={`${this.inputId}-${index}`}
```

Comment:
Yep, you're right! Meant to just update the id and accidentally also did `name`. Thanks!